README.md

```diff
@@ -344,6 +344,28 @@ a daily ubuntu-advantage-tools image.
 UACLIENT_BEHAVE_REUSE_IMAGE=your-custom-image-id tox -e behave-awspro-20.04
 ```
 
+### MOTD Messages
+
+Since ubuntu-advantage-tools is responsible for enabling ESM services, we advertise them on different
+applications thorough the system, such as MOTD and apt commands like upgrade.
+
+To verify that the MOTD message is advertising the ESM packages, ensure that we have ESM source list
+files in the system. If that is the case, please run the following commands to update the state of
+MOTD and display the message:
+
+```sh
+# Make apt aware of the ESM source files
+sudo apt update
```

Comment:
should we do a check for ua client version?

Reply:
I think we should assume we are running the latest version here, right ? Since this goes under the test section. But if need,  I can add that command, no problem at all